babel/dates.py

```diff
@@ -437,7 +454,10 @@ def get_timezone_location(dt_or_tzinfo=None, locale=LC_TIME):
                          the timezone; if `None`, the current date and time in
                          UTC is assumed
     :param locale: the `Locale` object, or a locale string
-    :return: the localized timezone name using location format
+    :param return_city: True or False
```

Comment:
The parameter description should include what it does too. Maybe something like

> Return only the exemplar city for the time zone when true

?